datadog/api/__init__.py

```diff
@@ -35,3 +35,7 @@
 from datadog.api.service_checks import ServiceCheck
 from datadog.api.tags import Tag
 from datadog.api.users import User
+from datadog.api.aws_integration import Aws
```

Comment:
Shouldn't we call classes `AWSIntegration` and Co instead. This kind of shortcuts can bit us later on.

Reply:
Thanks for the review! I'm happy to modify all of these new classes to have `Integration` suffixed on the end. Will update!